app/controllers/user_mentee_applications_controller.rb

```diff
@@ -61,4 +63,12 @@ def user_mentee_application_params
   def resume_params
     params.require(:user_mentee_application).permit(:resume, :resume_name)
   end
+
+  def send_notifications
+    UserMenteeApplication::ApplicationSubmissionNotification
+      .with(user_mentee_application: @user_mentee_application).deliver(@user_mentee_application.user)
+
+    UserMenteeApplication::ApplicationSubmissionAlert
+      .with(user_mentee_application: @user_mentee_application).deliver(User.admin)
```

Comment:
For the deliver, you can send to both Dave and I, that should be enough.

Reply:
lol:
```ruby
# user.rb

scope :dave, ~> { find_by(email: 'dave@aol.com') }
```
Probably better might be like, but we probably want some form of encapsulation for that, right?

```ruby
# user.rb

scope :team_leads, ~> { where(email: ['daniel@agencyoflearning.com','dave@agencyoflearning.com']) }
```

Reply:
Are these emails accurate? 😆  but yeas, i agree with this